chromium_src/ios/chrome/browser/sync/ios_chrome_sync_client.mm

```diff
@@ -14,10 +14,10 @@
 syncer::DataTypeController::TypeVector
 IOSChromeSyncClient::CreateDataTypeControllers(
     syncer::SyncService* sync_service) {
-  syncer::ModelTypeSet disabled_types = {syncer::PROXY_TABS, syncer::AUTOFILL,
-                                         // syncer::PREFERENCES,
-                                         syncer::READING_LIST,
-                                         syncer::USER_CONSENTS};
+  syncer::ModelTypeSet disabled_types = {
+      syncer::PROXY_TABS, syncer::AUTOFILL,
+      // syncer::PREFERENCES,
+      syncer::READING_LIST, syncer::SEND_TAB_TO_SELF, syncer::USER_CONSENTS};
```

Comment:
@Brandon-T Did you have a chance to test this manually?

Reply:
Yes, tested and it indeed disables send tab to self, even though Desktop doesn't obey the flags (but that's a separate issue).